### PR TITLE
Adding scale group monitoring to the monitoring role.

### PR DESCRIPTION
**Pull request by kwoodson**

### CODE REVIEW

**Review by joelddiaz — APPROVED** [review]

**Review by mwoodson — APPROVED** [review]

File: ansible/roles/oso_host_monitoring/tasks/scale_group.yml [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+        ansible-playbook /opt/config/configure_host.yml -v
+
+    openshift_daemonset_config_script: operations_config.sh
+    openshift_daemonset_config_secrets:
+    - path: docker-config.json
+      data: |
+        {{ lookup('template', '../templates/docker-config.json.j2') }}
+
+    openshift_daemonset_config_node_selector: "{{ osohm_sg_node_selector }}"
+        #kubernetes.io/hostname: ip-172-31-49-149.ec2.internal
```
Comment (joelddiaz) [inline ansible/roles/oso_host_monitoring/tasks/scale_group.yml]: extra stuff

File: ansible/roles/oso_host_monitoring/tasks/scale_group.yml [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+              dest: /opt/tmp_shared_config/monitoring_config.yml
+              owner: root
+              group: root
+
+      authorized_keys_tasks.yml: |
+          - name: setup authorized keys
+            copy:
+              remote_src: True
+              src: /opt/config/authorized_keys
+              dest: /host/root/.ssh/authorized_keys
```
Comment (joelddiaz) [inline ansible/roles/oso_host_monitoring/tasks/scale_group.yml:125]: add mode

File: ansible/roles/oso_host_monitoring/tasks/scale_group.yml [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+      authorized_keys: |
+        {% for user in osohm_authorized_keys | selectattr('pub_key', 'defined') %}
+        {{ user.pub_key }}
+        {% endfor %}
+
+      operations_config.sh: |
+        #!/bin/bash
+
+        #
```
Comment (joelddiaz) [inline ansible/roles/oso_host_monitoring/tasks/scale_group.yml]: extra line

**Review by joelddiaz — CHANGES_REQUESTED** [review]

File: ansible/roles/oso_host_monitoring/tasks/scale_group.yml (hunk 2 above)
Comment (kwoodson) [reply ansible/roles/oso_host_monitoring/tasks/scale_group.yml:125]: Added.  Its below

**Pull request merged**